oci/pkg/oci/bundle.go

```diff
@@ -83,8 +84,8 @@ func readPath(filePath string) ([]ParsedTektonResource, error) {
 		return nil, err
 	}
 
-	// Try to tease out the type of the file from the extension and load them into a slice (if there are multiple
-	// entities in a single file).
+	// Try to tease out the type of the file from the extension and load them
+	// into a slice (if there are multiple entities in a single file).
```

Comment:
[nit] I don't believe 80 chars is enforced elsewhere is it?

Reply:
agree, it isn't enforced and I agree we shouldn't enforce personal prefs as well .. It was reading the code and the edit was just part of how I was understanding it. ... I guess the existing comment already consumed two lines :)